packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -116,19 +111,31 @@ async function downloadGoCardlessTransactions(
     throw BankSyncError(res.error_type, res.error_code);
   }
 
-  const {
-    transactions: { all },
-    balances,
-    startingBalance,
-  } = res;
+  if (includeBalance) {
+    const {
+      transactions: { all },
+      balances,
+      startingBalance,
+    } = res;
 
-  console.log('Response:', res);
+    console.log('Response:', res);
 
-  return {
-    transactions: all,
-    accountBalance: balances,
-    startingBalance,
-  };
+    return {
+      transactions: all,
+      accountBalance: balances,
+      startingBalance,
+    };
+  } else {
+    const {
+      transactions: { all },
+    } = res;
+
+    console.log('Response:', res);
+
+    return {
+      transactions: all,
+    };
```

Comment:
Is it really necessary to destructure, then immediately restructure here?  Couldn't you just return `res`?

Reply:
This one's probably unnecessary.